prime-dotnet-webapi/Services/interfaces/IOrganizationService.cs

```diff
@@ -25,5 +28,13 @@ public interface IOrganizationService
         Task<SignedAgreementDocument> AddSignedAgreementAsync(int organizationId, int agreementId, Guid documentGuid);
         Task<SignedAgreementDocument> GetLatestSignedAgreementAsync(int organizationId);
         AgreementType OrgAgreementTypeForSiteSetting(int careSettingCode);
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="organizationId"></param>
+        /// <param name="newSigningAuthorityId"></param>
+        /// <returns><c>true</c> if can successfully switch signing authority, <c>false</c> otherwise.</returns>
+        Task<bool> SwitchSigningAuthorityAsync(int organizationId, int newSigningAuthorityId);
```

Comment:
I would prefer the comment be in the comment body, rather than in the return value of the Method